alto/commands/cromwell/run.py

```diff
@@ -1,8 +1,7 @@
-import argparse, getpass, json, os, requests, time
+import argparse, getpass, json, os, requests, time, sys, zipfile
```

Comment:
If you agree with using `raise Exception` to replace `sys.exit`, then you can remove `sys` import here.